client/src/lib/navigation/Stacks.tsx

```diff
@@ -28,7 +28,11 @@ const TempleStackWrapper = () => (
       }}>
       <TempleStack.Screen name={'ChangingRoom'} component={ChangingRoom} />
       <TempleStack.Screen name={'Portal'} component={Portal} />
-      <TempleStack.Screen name={'Temple'} component={Temple} />
+      <TempleStack.Screen
+        options={{gestureEnabled: false}}
```

Comment:
Pretty sure that this won't disable the hardware back-button on Android which would lead to the same issue as swiping back. May I suggest that we don't have this as a separate route and instead render the portal on top of call until it's started?

Reply:
Great point, let's do that instead. Thanks @gewfy 🏓

Reply:
how is that going to fix anything? won't you be able to navigate back from the portal anyways?
maybe I misunderstood what the issue is... 🤔

Reply:
navigate back -> join portal again 🤷 

what are we trying to fix exactly?